### ISSUE TITLE
`gaia` doesn't terminate after force quitting the app.

### ISSUE THREAD

**Issue by nylira** [cosmos/cosmos-ui#102]
@mappum  when I force quit `npm run testnet` with CMD+C in the terminal, `gaia` appears to keep running. I have to find `gaia` with `ps aux | grep gaia` and kill it there before I can restart the app.

```
electron  spawning /Users/pz/go/bin/gaia with args ""server" "serve" "--home" "/Users/pz/.cosmos-ui-dev/baseserver""

electron  gaia: 2017/11/22 14:13:06 Serving on ":8998"

electron  gaia: ERROR: listen tcp :8998: bind: address already in use

electron  gaia exited with code 1

electron  baseserver crashed, restarting

electron  startBaseserver /Users/pz/.cosmos-ui-dev/baseserver

electron  spawning /Users/pz/go/bin/gaia with args ""server" "serve" "--home" "/Users/pz/.cosmos-ui-dev/baseserver""

electron  gaia: 2017/11/22 14:13:07 Serving on ":8998"

electron  gaia: ERROR: listen tcp :8998: bind: address already in use

electron  gaia exited with code 1

electron  baseserver crashed, restarting

^C⏎                                                                                      
pz@mbp ~/P/cosmos-ui> ps aux | grep gaia
pz               69589   0.0  0.0  4267752    780 s008  R+    2:13PM   0:00.00 grep --color=auto gaia
pz               69022   0.0  0.1 558464976   8568 s008  S     2:09PM   0:00.03 /Users/pz/go/bin/gaia server serve --home /Users/pz/.cosmos-ui-dev/baseserver
pz@mbp ~/P/cosmos-ui> kill 69022
pz@mbp ~/P/cosmos-ui> npm run testnet
```

**Comment by faboweb** [comment]
relates to #62 #63

**Comment by nylira** [comment]
Same as #62

**Comment by jolesbi** [comment]
someone has to say closes #62 for it to close
